package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mendix/pluggable-widgets-tools",
-  "version": "0.1.1",
+  "version": "0.2.0",
```

Comment:
Please execute `npm install` to fix package-lock :)

Reply:
Yess!!

Reply:
Thankss